src/ui/surfaces/SiteConfiguration.tsx

```diff
@@ -98,22 +144,16 @@ function EventSettings() {
       <Form
         className="tw-pt-6 tw-max-w-170"
         defaultValues={query.data ?? {
-          collectAutomatedEvents: true,
           collectDownloads: true,
```

Comment:
I wonder why collectPageViews:true, but the checkbox was not selected by default. Do you know why?